### ISSUE TITLE
Godot crashing when selecting multiple visual scritping nodes including a function node and then deleting them

### ISSUE THREAD

**Issue by xxSeys1** [godotengine/godot#64662]
### Godot version

3.5 stable

### System information

Linux Mint Cinnamon 21

### Issue description

Godot is constantly crashing when selecting multiple visual scripting nodes including a function node(!) and then deleting them.

### Steps to reproduce

Add an (inbuild) function to a script
add another node
mark the nodes (including the function!)
delete them 

### Minimal reproduction project

[VS Issue.zip](https://github.com/godotengine/godot/files/9387731/VS.Issue.zip)

**Comment by amoriqbal** [comment]
Verified on branch 3.5
Here is the stack trace.
```
ERROR: FATAL: Index p_index = 2 is out of bounds (size() = 2).
   at: get (./core/cowdata.h:150)

================================================================
handle_crash: Program crashed with signal 4
Engine version: Godot Engine v3.5.1.rc.custom_build (1ea8b9d2b7da2285e890e087ff2b354e4d458b66)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x43090) [0x7fac84223090] (??:0)
[2] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4971b9a] (/media/amor/DATA/Projects/godot3_x/./core/cowdata.h:150)
[3] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x2630392] (/media/amor/DATA/Projects/godot3_x/modules/visual_script/visual_script_editor.cpp:1850)
[4] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x269c1ad] (/media/amor/DATA/Projects/godot3_x/./core/method_bind.gen.inc:?)
[5] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4a3b882] (/media/amor/DATA/Projects/godot3_x/core/object.cpp:?)
[6] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4a3db08] (/media/amor/DATA/Projects/godot3_x/./core/variant.h:437)
[7] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4a3c0b8] (/media/amor/DATA/Projects/godot3_x/core/object.cpp:1285)
[8] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x39e332c] (/media/amor/DATA/Projects/godot3_x/scene/gui/graph_edit.cpp:1384)
[9] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x229d530] (/media/amor/DATA/Projects/godot3_x/./core/reference.h:257)
[10] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4a3a340] (/media/amor/DATA/Projects/godot3_x/./core/variant.h:437)
[11] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4a3b719] (/media/amor/DATA/Projects/godot3_x/core/object.cpp:864)
[12] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x390d9b6] (/media/amor/DATA/Projects/godot3_x/scene/main/viewport.cpp:2404)
[13] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x390990d] (/media/amor/DATA/Projects/godot3_x/./core/reference.h:257)
[14] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x3909608] (/media/amor/DATA/Projects/godot3_x/./core/reference.h:257)
[15] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x229d530] (/media/amor/DATA/Projects/godot3_x/./core/reference.h:257)
[16] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4a3b882] (/media/amor/DATA/Projects/godot3_x/core/object.cpp:?)
[17] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x4a3b65e] (/media/amor/DATA/Projects/godot3_x/core/object.cpp:849)
[18] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x38d5e8b] (/media/amor/DATA/Projects/godot3_x/scene/main/scene_tree.cpp:304)
[19] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x38d7274] (/media/amor/DATA/Projects/godot3_x/scene/main/scene_tree.cpp:463)
[20] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x1ba0235] (/media/amor/DATA/Projects/godot3_x/./core/reference.h:257)
[21] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x1ba18e7] (/media/amor/DATA/Projects/godot3_x/./core/list.h:278)
[22] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x1b85eda] (/media/amor/DATA/Projects/godot3_x/./core/local_vector.h:309)
[23] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x1b8ba01] (/media/amor/DATA/Projects/godot3_x/platform/x11/os_x11.cpp:3985)
[24] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x1b79105] (/media/amor/DATA/Projects/godot3_x/platform/x11/godot_x11.cpp:61)
[25] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7fac84204083] (??:0)
[26] /media/amor/DATA/Projects/godot3_x/bin/godot.x11.opt.tools.64.llvm() [0x1b78fae] (??:?)
-- END OF BACKTRACE --
================================================================
```

**Comment by akien-mga** [comment]
According to #64760 this is a regression in 3.5 (doesn't happen in 3.4.5).

**Comment by akien-mga** [comment]
It's a bug introduced in 32b9818965e3ee32db7ef43e9e362b0a6e0c2aff @Chaosus. From a quick look it seems like you made a typo in the VisualScript implementation, it's checking `graph->get_child_count()` and then indexing `p_nodes`. (It should check `p_nodes->get_child_count()`.)

**Comment by akien-mga** [comment]
Fixed by #64772.
